js/bybit.js

```diff
@@ -5133,38 +5630,47 @@ module.exports = class bybit extends Exchange {
         const toId = this.safeString (accountTypes, toAccount, toAccount);
         const currency = this.currency (code);
         const amountToPrecision = this.currencyToPrecision (code, amount);
-        const request = {
-            'transfer_id': transferId,
-            'from_account_type': fromId,
-            'to_account_type': toId,
-            'coin': currency['id'],
-            'amount': amountToPrecision,
-        };
-        const response = await this.privatePostAssetV1PrivateTransfer (this.extend (request, params));
+        let method = undefined;
```

Comment:
@ttodua We're missing a couple of things here (correct link, camelCase params instead of snake_case), can you please check @Dan-krm's PR and apply the same changes here? https://github.com/ccxt/ccxt/pull/15579/files#diff-cb8b530c783df8bccc6e0df910e7433b34c9c9c3cea8904fcf2c59f1e60aeebe

Reply:
I've accommodated all changes from @Dan's PR and corrected/updated those methods too.
I think after merge, i'll just edit some peripherical thins (i.e. links etc...), however, because of the priority, I don't want to hold this PR for some minor things.